README.md

```diff
@@ -106,6 +106,11 @@ aggregate?: (expression: Alg.AggregateExpression) => Promise<RDF.Term>;
 
 You can probably ignore this.
 
+We also provide an `AsyncAggregateEvaluator` to that works the same way `AggregateEvaluator` does.
+The signature of only the `put` method changes to be async. It is up to you to handle this correctly.
+You are for example expected to await all puts before you ask for `result`.
+You should also note the order of calling and awaiting put while using the `GroupConcat` aggregator.
```

Comment:
`GroupConcat` doesn't exist in Sparqlee, to this comment does not make much sense here.

Reply:
It exists in Sparqlee: https://github.com/comunica/sparqlee/blob/6c609b165e48bb190042b1fdbaf019e2c78d59f5/lib/evaluators/AggregateEvaluator.ts#L266

Reply:
Nvm my comment then, I thought it was part of Comunica.
See, that's how well I know how the expressions work 😅